OpenSim/Common/C3DFileAdapter.h

```diff
@@ -44,10 +42,13 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     C3DFileAdapter& operator=(C3DFileAdapter&&)      = default;
     ~C3DFileAdapter()                                = default;
 
+    explicit C3DFileAdapter(const bool& useSIMMColumnOrdering);
```

Comment:
I don't think a `C3DAdapter` or any `FileAdapter` other than the one that writes SIMMMotion (.mot) files should be aware of `useSIMMColumnOrdering`.  If the goal is to produce *.mot* files that can be read in by SIMM and older versions of OpenSim then the flag should be on `STOFileAdapter::write()` to write as a *.mot*. We do not want anything within OpenSim to be dependent on any ordering specified by another package.